content/en/blog/2025/upgrade_from_3.5_to_3.6_issue.md

```diff
@@ -72,3 +75,4 @@ test via [19634][], which was also backported to release-3.6 via [19662][].
 [19636]: https://github.com/etcd-io/etcd/pull/19636
 [19634]: https://github.com/etcd-io/etcd/pull/19634
 [19662]: https://github.com/etcd-io/etcd/pull/19662
+[[Follow-up] Preventing Upgrade Failures from etcd v3.5 to v3.6][]: link-to-new-blog
```

Comment:
Looks like we'd better do it separately. Otherwise the workflow check fails. @jberkus

Reply:
remove the change to the existing blog...

Will update it after this new blog is updated.